tests/backup/backup_helper.go

```diff
@@ -2672,6 +2672,13 @@ func upgradeStorkVersion(storkImageToUpgrade string) error {
 			return err
 		}
 		storageSpec.Spec.Stork.Image = finalImageToUpgrade
+
+		// Check to reset customImageRegistry to blank as in case of ibm it'll be icr.io/ext/ and not
+		// docker.io/ which causes issues when we try to install stork which is not pushed to icr.io/ext
+		if os.Getenv("CLUSTER_PROVIDER") == "ibm" {
```

Comment:
Is it possible to use GetClusterProviders()?

Reply:
Done